src/actions/guides/database/database_setup.cr

```diff
@@ -0,0 +1,113 @@
+class Guides::Database::DatabaseSetup < GuideAction
+  guide_route "/database/database-setup"
+
+  def self.title
+    "Database Setup"
+  end
+
+  def markdown
+    <<-MD
+    ## Configure
+
+    Ensure that your system is running a [PostgreSQL](https://www.postgresql.org/docs/current/tutorial-install.html) version of at least 9.4 or later.
+
+    > If you're using the postgres app on macOS, be sure to install [CLI tools](https://postgresapp.com/documentation/cli-tools.html).
+
+    ### Standard Options
+
+    To configure Lucky to connect to your database, you can define a `DATABASE_URL` environment variable. (e.g. `postgres://root@localhost:5432/my_database`). To see more options, look in your app's `config/database.cr` file.
+    In this file, you'll find options for configuring
+
+    * database_name
+    * hostname
+    * username
+    * password
+    * port
+
+    ### Connection Pool
+
+    Lucky also supports connection pool settings:
+
+    * initial_pool_size
+    * max_pool_size
+    * max_idle_pool_size
+    * checkout_timeout
+    * retry_attempts
+    * retry_delay
+
+    To set the connection pool options, just append a query string to the end of the `settings.url` option (e.g. `?initial_pool_size=5&retry_attempts=2`)
+
+    ### Avram Configuration
+
+    Avram requires a `url` option to be set. If you decide to not use Avram as your ORM, you can set this option to a string like `"noop"`.
```

Comment:
I’ve seen lots of people get confused by “noop” (reading it like poop 💩). Thoughts on “unused”?